static/templyte/src/app/download-template/download-template.component.ts

```diff
@@ -42,9 +49,12 @@ export class DownloadTemplateComponent implements OnInit {
                     component.ngZone.run(() => { // Need to do this using NgZone since we're calling a third party API
                         component.templateDirectoryInfoRef = component.db.object('template-directory/' + params.id);
                         component.templateRenderInfoRef = component.db.object('template-render-info/' + params.id);
+                        component.templateRatingsInfoRef = component.db.object('template-ratings/' + params.id);
+                        component.templateRatingsInfoDatabaseRef = firebase.database().ref('template-ratings/' + params.id);
+                        component.templateDirectoryInfoDatabaseRef = firebase.database().ref('template-directory/' + params.id);
```

Comment:
The ones that are refs are named a little strangely - you can rename the variables we already have to something more appropriate if you need to

Reply:
The names seem clear to me, so I am going to mark this as resolved. We can change them later if you still find them unclear.